Tasks/MavenAuthenticateV0/task.json

```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 165,
+        "Minor": 166,
```

Comment:
The change in the PR seems reasonable to me. Although, could you please change the minor version update into 167? We have an internal system for minor version updates.

Reply:
For sure, done!